mlflow/recipes/utils/wrapped_recipe_model.py

```diff
@@ -8,6 +8,8 @@
 
 
 class WrappedRecipeModel(PythonModel):
+    _skip_wrapping_predict = True
```

Comment:
Same as above - the predict function does not have type hint so it should be no-op not skipping the wrapping?

Reply:
Yes it's no-op, I can remove this. Just if we add type hints in the future (e.g. even Any) then we'll have some data validation